.github/dependabot.yml

```diff
@@ -1,36 +1,52 @@
 version: 2
+
 updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "monday"
+      time: "05:00"
```

Comment:
I would make it earlier in the morning so that we get a chance to talk about them in the meetings in case of any issues

Reply:
5am looked soon enough to me. What time do you suggest?

Reply:
Oh my bad, I use the AM/PM system 😅